core/src/main/java/com/orientechnologies/orient/core/storage/impl/local/paginated/wal/OWALPage.java

```diff
@@ -30,10 +31,29 @@
   public static final int  PAGE_SIZE       = OGlobalConfiguration.DISK_CACHE_PAGE_SIZE.getValueAsInteger() * 1024;
   public static final int  MIN_RECORD_SIZE = OIntegerSerializer.INT_SIZE + 3;
 
-  public static final  int CRC_OFFSET          = 0;
-  public static final  int MAGIC_NUMBER_OFFSET = CRC_OFFSET + OIntegerSerializer.INT_SIZE;
+  public static final int CRC_OFFSET          = 0;
+  public static final int MAGIC_NUMBER_OFFSET = CRC_OFFSET + OIntegerSerializer.INT_SIZE;
   public static final int FREE_SPACE_OFFSET   = MAGIC_NUMBER_OFFSET + OLongSerializer.LONG_SIZE;
-  public static final  int RECORDS_OFFSET      = FREE_SPACE_OFFSET + OIntegerSerializer.INT_SIZE;
+
+  /**
+   * Information about position of LSN of last record which end is stored on this page. If only begging of the record is stored on
+   * this page its position is not stored.
+   *
+   * @see OLogSegment.FlushTask#commitLog()
+   * @see OLogSegment#init(ByteBuffer)
+   */
+  public static final int LAST_STORED_LSN = FREE_SPACE_OFFSET + OIntegerSerializer.INT_SIZE;
```

Comment:
Seems like the layout of the WAL page is changed. Will it be safe/possible to use the new version on the old WAL?